articles/cosmos-db/sql-api-get-started.md

```diff
@@ -64,6 +64,7 @@ Let's create an Azure Cosmos DB account. If you already have an account you want
 5. In the **NuGet** tab, click **Browse**, and type **azure documentdb** in the search box.
 6. Within the results, find **Microsoft.Azure.DocumentDB** and click **Install**.
    The package ID for the Azure Cosmos DB SQL API Client Library is [Microsoft Azure Cosmos DB Client Library](https://www.nuget.org/packages/Microsoft.Azure.DocumentDB/).
+   **Note**: If you are using .NET Core, please see [the .NET Core docs](https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-dotnetcore-get-started).
```

Comment:
@TomasHubelbauer we use inline links for docs in the same repo. Can you update the link to `[the .NET Core docs](sql-api-dotnetcore-get-started.md) instead